search_gov_crawler/search_gov_spiders/utility_files/crawl-sites.json

```diff
@@ -0,0 +1,332 @@
+[
+    {
+        "allowed_domains": "armymwr.com",
+        "handle_javascript": false,
```

Comment:
nit: Can we just have:
```
"handle_javascript": true,
```

Where needed and no field at all if it doesn't ex:
```json
{
        "allowed_domains": "travel.dod.mil",
        "starting_urls": "https://www.travel.dod.mil/"
},
{
        "allowed_domains": "help.nfc.usda.gov",
        "handle_javascript": true,
        "starting_urls": "https://help.nfc.usda.gov/"
},
```
But, this is also fine